server/events/hullPlating.ts

```diff
@@ -0,0 +1,52 @@
+import App from "../app";
+import { pubsub } from "../helpers/subscriptionManager";
+import uuid from "uuid";
+import throttle from "../helpers/throttle";
+import { HullPlating } from "../classes";
+
+const sendUpdate = throttle(() => {
```

Comment:
I don't think throttling this is strictly necessary, but it also won't cause any major problems as far as I can tell.